lib/origami-image-set-tools.js

```diff
@@ -16,6 +19,8 @@ module.exports = class OrigamiImageSetTools {
 	constructor(options) {
 		this.options = defaults({}, options, module.exports.defaults);
 		this.log = this.options.log;
+		this.scheme = this.options.scheme;
+		this.version = (semver.valid(this.options.version) ? this.options.version : 'v0.0.0');
```

Comment:
What do you think about having the `else` case pulling it's value from `module.exports.defaults.version`?